rts/Game/UI/KeyBindings.cpp

```diff
@@ -832,6 +846,34 @@ bool CKeyBindings::ExecuteCommand(const std::string& line)
 			debugEnabled = atoi(words[1].c_str());
 		}
 	}
+	else if (command == "keyload") {
+		const std::string filename = words.size() > 1 ? words[1] : "uikeys.txt";
```

Comment:
Can these be made const references? I am unsure how the lifetime extension for temporaries for the "uikeys.txt" case would work here, but it would avoid the need for a copy for the other case.

Reply:
You mean `const type name const`?

I feel it might not be too relevant to care about optimization in this section, but if you think it can be done sure.

Reply:
I mean `&`, as in `const std::string &filename`. But yes, this is a microoptimisation that maybe a smart compiler would even be able to do on its own. If you're not comfortable with it then it's probably best to leave it working as is.

Reply:
I see, passing by reference. Sure, can do that :+1: